platform/email/src/test/java/org/motechproject/email/web/SettingsControllerTest.java

```diff
@@ -35,12 +36,15 @@
 
     private static final String HOST = "localhost";
     private static final String PORT = "8099";
+    private static final String USERNAME = "user";
+    private static final String PASSWORD = "8099";
     private static final String LOG_ADDRESS = "true";
     private static final String LOG_SUBJECT = "true";
     private static final String LOG_BODY = "true";
     private static final String LOG_PURGE = "true";
     private static final String LOG_TIME = "1";
     private static final String LOG_MULTIPLIER = "weeks";
+    private static final Properties ADDITIONAL_PROPERTIES = new Properties();
```

Comment:
Can we have at least one test that actually has got those properties set and verifies they are properly set?